tgui/.eslintrc.yml

```diff
@@ -1,6 +1,7 @@
-parser: '@babel/eslint-parser'
+root: true
+parser: '@typescript-eslint/parser'
 parserOptions:
-  ecmaVersion: 2019
+  ecmaVersion: 2020
```

Comment:
🤔

Reply:
https://www.freecodecamp.org/news/javascript-new-features-es2020/

Reply:
it's 2021 u doofus

Reply:
I'm porting a PR, not getting us to the bleeding edge.

https://github.com/tgstation/tgstation/blob/master/tgui/.eslintrc.yml#L4